server/static/bindata_assetfs.go

```diff
@@ -179,47 +183,127 @@ func staticImagesAtlantisIcon_512Png() (*asset, error) {
\x4f\x80\x75\x4e\x37\x68\x03\xa2\x85\xcd\x40\x55\xbe\x2f\x17\x24\x06\xfe\xf9\xf1\x7a\x64\x87\xaa\xf0\x59\x64\xde\x1b\x20\xe2\xc4\xed\x62\xf9\xc5\xc3\xbb\xf8\x46\x82\xa6\x1c\x19\x81\x1e\xc2\xe0\x9e\x47\xd8\xf7\x93\x6e\xc9\xcb\x2c\x2e\x4b\x59\x56\xae\xca\xf0\x9b\x3f\x6d\xcd\x96\x94\xa3\x21\xbc\x49\x37\x28\x99\xfe\x5e\xc6\xbe\x3f\xfa\x36\x14\x72\xff\x45\xf2\x10\x7e\x5b\xd7\xa3\xdb\xa9\x60\xa5\x90\xfd\xf2\x7d\x04\x0b\xd1\x0a\x98\x47\xa7\x77\x95\x5a\xb9\x35\x15\x12\x7e\x88\x3c\x8e\xca\x05\xfc\xd1\x54\x56\xd7\x7f\xd8\x7a\xf1\x01\x6d\x28\x63\xe6\xde\xf7\x1f\x18\x62\x8e\xdc\x1c\xd7\x35\x93\xc7\x77\x26\xa7\x02\xd8\xc0\xba\x46\xa8\x54\x8b\xdc\xca\xe0\xbd\x34\xf1\x30\x5e\x96\xb4\xb4\x12\x8e\x82\x91\x94\x61\x89\x87\xfa\x05\x49\x49\xaf\x30\x26\x39\x45\x19\xdd\xca\x4e\x18\x26\x58\x6d\x9c\x7c\x32\xc1\x59\x98\x47\x14\x95\x4b\xef\x6f\xde\xb8\x0c\xbc\x00\x5a\xf6\x00\x39\x8d\xef\x19\x92\xaf\xf1\x62\x4d\xb3\xe9\x6f\x79\xca\x91\x47\x3c\xdc\x48\x34\x71\x34\xf5\x9b\x29\x88\xab\xaf\x80\x5a\xe5\xc5\x8b\x2c\x43\x6b\x98\x74\x8b\x03\xbe\x45\x82\x8c\x66\xb8\xd9\xa4\x3c\xce\xb2\x87\x43\x49\x29\xbd\x92\xab\xab\x14\x06\xbd\x21\x36\x4d\x63\xcf\x6a\xc2\x0e\xf5\x09\xf1\x9e\xcc\x3d\xac\xb7\x71\xbb\xb7\xe5\x29\xe3\x60\x4e\xaf\x86\xa4\xd8\xb7\x72\xe3\xb7\xe7\x6e\x7d\x5e\x85\xf5\xc7\x9f\x5f\x4f\xd7\xf1\x7a\xcb\xbe\x86\x79\xf1\xfd\x84\x65\x4c\xb0\x33\x16\x16\xd3\x72\x9b\x6e\x04\xc2\x11\x61\x1a\x56\x28\x77\xb0\x89\xc4\x3b\xed\x41\x26\xbc\x8a\xe8\x68\x46\x58\xfb\x7e\xcd\x5a\x01\xe9\x65\x5f\x4d\x63\x11\xb6\xc2\xb9\x1a\xaf\x8f\xe4\xbe\x68\xe7\xcb\x08\x66\xec\x84\x09\x07\x5a\x7c\x5f\x9c\xd2\xb9\x08\x4c\x04\x95\x07\x61\xa7\xb7\x1b\xd6\xc5\xa1\xfa\x48\x59\x7b\x98\x14\x94\x77\xa1\xa2\x98\x4c\xf0\xf5\x34\x16\xa2\xf8\x2a\xe6\x49\xc6\x42\x1e\x16\x51\x44\x45\x5b\xdb\xbe\x53\x9b\xf0\x7d\x26\x6b\xf1\xfd\xb9\xa5\x3f\x12\x5d\xaa\x7b\xe1\xdc\xb3\x69\x99\x57\xc5\x9a\xbd\xe1\x09\xbb\x9f\x08\xf7\x4e\xe2\x82\xc2\x6c\xe8\x02\xf0\x2c\x56\xdd\xe1\x94\x4f\x25\xa1\xba\x4a\xaf\xb3\x94\xdf\x80\xc8\xd3\x39\xb4\x4d\xe6\x56\xc6\xb1\x9c\x07\x93\x79\xdb\xcb\x44\x4e\xe7\xa1\x07\x0b\x8e\xca\x0f\xd8\xab\xc7\x48\xa6\xec\x30\xa8\x62\x28\x15\xce\x5c\x6e\x99\xa3\x7a\x38\x96\x86\x9f\xaa\x4d\x77\x13\xb5\xed\x8a\xba\xf6\x14\x37\x07\x77\x6e\x7b\x2e\xa4\xdd\xfc\xc9\x38\x36\x79\xb1\x03\x05\xd5\x92\x75\x00\x64\x34\xef\xf0\x09\x4b\x2f\x8b\xaf\x59\xa6\x4a\x3a\xd7\xce\x37\x9d\x0a\xec\x87\xb2\x6f\xc1\xd1\x6d\x5a\xbe\x74\x1e\xd4\xb5\xfb\x64\x44\xe9\x48\xf8\x7e\x2c\xb7\xc0\xd0\xd7\x4e\xeb\x72\xcc\xee\x3b\x67\xdc\xb7\x0c\xc5\x76\xdc\x99\xc3\x09\xe5\xf6\x69\x4e\xc7\x39\x71\x5f\x75\xa4\x8e\x31\xa8\x93\x0d\xbf\x90\x63\x92\xd2\xa2\x0b\xea\xe9\x64\x82\x59\xc8\x69\x11\xa6\x91\x24\x05\x20\x10\x18\x21\x21\x7f\xe4\x35\xc6\x8d\xfc\xdf\x76\xe9\xa1\xb3\xe9\x7d\x7f\x48\xd3\x3e\x4c\xf1\x7c\x9f\x35\x9b\xbc\x40\xec\x2c\xe5\x67\x09\x2d\xd9\x54\x4b\x9d\x28\x88\xc4\x4b\x39\x7d\x3f\x7f\xf3\x35\x1d\x84\xa7\x78\xc7\xca\x7d\xbc\x66\x3f\xbe\x7d\x43\x38\x45\x3d\x2e\x45\xd2\x0d\x2b\x8e\xd1\x0d\x1b\x61\xee\x2f\x9a\xdc\xd4\x35\xf7\x7d\x6e\x21\xb3\xae\x3d\x79\xd6\x90
```

Comment:
Is there a reason why we can't read this in from a separate file?

Reply:
This is a generated file
```
// Code generated for package static by go-bindata DO NOT EDIT. (@generated)
```